setup.py

```diff
@@ -75,7 +75,7 @@ def find_version(package):
         'OpenCL': ["pyopencl"],
         'CUDA': ["pycuda"],
     },
-    build_requires=['setuptools'],
-    test_requires=['pytest'],
-    cmdclass = {'test': PyTest},
+    #setup_requires=['setuptools'],
+    tests_require=['pytest'],
```

Comment:
Hmm... running `python setup.py test` should show this typo earlier, are you running unit tests during Travis builds?

Reply:
Tests run during travis builds use the environment that we set up, which includes pytest.